ofl/playwritero/article/ARTICLE.en_us.html

```diff
@@ -0,0 +1,151 @@
+<p>
+ The curriculum goals for
+ <i>
+  Comunicare în limba română
+ </i>
+ , or Communication in the Romanian language, place importance on teaching handwriting.
+ Instruction happens during the early years of primary education, specifically in preparatory, first, and second grades.
+ While schools and teachers have some flexibility in their methods, the instruction process, its stages, and writing models are widely standardized.
+</p>
+<p>
+ Handwriting education begins in preparatory grade with a focus on print script alphabet, also known as
+ <i>
+  ‘scris de tipar‘
+ </i>
+ or
+ <i>
+  ‘litere bloc’
+ </i>
+ , and on development of communication skills.
+ In first grade, students gradually transition to a cursive writing model called
+ <i>
+  ‘litere de mână‘
+ </i>
+ .
+ It is a continuous cursive style with a 18º slope, and its narrow proportions follow a grid.
+</p>
+<p>
+ Playwrite România is a variable font with a weight range from Thin (100) to Regular (400), and supports over 150 Latin-based languages.
+</p>
+<p>
+ To contribute, see
+ <a href="https://github.com/TypeTogether/Playwrite/">
+  github.com/TypeTogether/Playwrite
+ </a>
+ .
+</p>
+<hr/>
+<img src="Playwrite-Romania-1.png"/>
+<h4>
+ Playwrite România characteristics
+</h4>
+<p>
+ This slanted continuous cursive features a slow stroke speed and medium-length looped ascenders and descenders.
+ The uppercase letters are decorative, highlighted by a straight spine in 'S' and a distinguishable shape in 'G'.
+ Both capital and lowercase versions of 'X' and 'Z' include crossbars as a unifying stylistic element.
+ The lowercase letters often incorporate knots, enhancing the complex appearance of the script.
+ The letter 'f' has a loopless descender and connects to the next letter via its crossbar, while 'q' is notable for a short exit stroke in its descender.
+</p>
+<img src="Playwrite-Romania-3.png"/>
+<p>
+ Playwrite Romania appears in font menus with a two-letter country code ‘RO’,
```

Comment:
Should Romania have a accent here? If I compare with other PW description, it seems it uses the local language in the description (english is only for the link, right?)

Reply:
We are localizing it everywhere it appears as the font name. For example, in this subheading.
I've updated it now.